WISE-Community/WISE
# Added banner to beta site

Added a banner announcing new site above the header. Will change the message/action when we switch over permanently on production.

To test:
- Load site and make sure banner displays correctly.
- Select "Learn More" link in the banner and test out the two buttons in the dialog that appears as well as the 'Contact us' link.
- Click the close button to the right of the banner. The banner should disappear and the rest of the page content should adjust position accordingly.

Closes #1614.
